sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -1350,6 +1350,14 @@ void CurlConnectionPool::CleanUp()
       std::this_thread::sleep_for(
           std::chrono::milliseconds(_detail::DefaultCleanerIntervalMilliseconds));
 
+      // while sleeping, it is allowed to explicitly prevent the cleaner to run and stop it, for
+      // example, when the application exits, and the cleaner is sleeping, we don't want it to wake
+      // up and try to access de-allocated memory.
+      if (!CurlConnectionPool::s_isCleanConnectionsRunning)
```

Comment:
We don't have `s_` prefix. We use `g_` for globals and statics.

Reply:
done